__tests__/app.test.js

```diff
@@ -50,4 +50,74 @@ describe('APP be-nc-news', () =>
             })
         })
     })
-})
\ No newline at end of file
+    describe('GET /api/articles/:article_id', () =>
+    {
+        test('GET: will return with a status of 200', () =>
+        {
+            return request(app).get('/api/articles/1').expect(200)
+        })
+        test('responds with an object that isn\'t empty', () =>
```

Comment:
nice incremental TDD. But maybe too incremental! 
This test is made redundant by the next one.